lms/static/sass/base/_variables.scss

```diff
@@ -82,6 +82,28 @@ $gray-d2: shade($gray,40%); // #4c4c4c
 $gray-d3: shade($gray,60%); // #323232
 $gray-d4: shade($gray,80%); // #191919
 
+// TO-DO: once existing lms $blue is removed, change $cms-blue to $blue. 
```

Comment:
Not sure if this is a holdover from an earlier PR, but I think it's better to clean up the existing blues instead of wiring in a whole new set, especially since we have made the base LMS blue a11y-friendly.